### DISCUSSION TITLE
My achievements

### DISCUSSION THREAD

**Discussion by Kasunm12** [community/community#18188] [Other Feature Feedback, Questions, & Ideas]
There's a new feature on my profile that shows achievements such as  Pull Shark.

Maybe also a way to toggle them would be nice as well. And your ideas?

**Comment by wopian** [comment]
At the bottom of your [profile settings](https://github.com/settings/profile) there is a checkbox to turn them off.

<img width="415" alt="Screenshot 2022-06-10 at 13 40 41" src="https://user-images.githubusercontent.com/3440094/173066226-3a1b73af-4449-49db-a5cd-7ed3e11b33fa.png">
